Earthfile

```diff
@@ -105,7 +105,7 @@ test-integration:
   COPY +go/translate /translate
   COPY --dir migrate/mysql migrate
   WITH DOCKER --compose compose.yaml --service mysql --pull migrate/migrate:v$migrate_version --pull golang:$go_version-alpine
-    RUN \
+    RUN --secret=googletranslate_api_key \
```

Comment:
TRANSLATE_TRANSLATE_SERVICES_GOOGLE_TRANSLATE_API_KEY is unsafely passed to docker run as an environment variable. I don't know what the implications are but let's be on the safe side. I hope it is not saved in the BuildKit layers.

Use `RUN --no-cache`.

Reply:
It would be nice to find out from earthly

Reply:
Tried to find something about that, but no luck with that. For now, will stick with `--no-cache` (in **ci/cd** pipeline it does not matter anyways as every run is run in sandbox environment, this will only affect local integration test execution speed, when running `earthly +test-integration` )